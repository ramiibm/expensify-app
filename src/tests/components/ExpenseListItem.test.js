import React from 'react';
import { shallow } from 'enzyme';

import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('should render ExpenseListItem with expenses', () => {
  expenses.map(expense => {
    const wrapper = shallow(<ExpenseListItem key={expense.id} {...expense} />);
    expect(wrapper).toMatchSnapshot();
  });
});
