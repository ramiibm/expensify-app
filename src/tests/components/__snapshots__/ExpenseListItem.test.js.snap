// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expenses 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    195
     - 
    0
  </p>
</div>
`;

exports[`should render ExpenseListItem with expenses 2`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    109500
     - 
    -345600000
  </p>
</div>
`;

exports[`should render ExpenseListItem with expenses 3`] = `
<div>
  <Link
    to="/edit/3"
  >
    <h3>
      Credit Card
    </h3>
  </Link>
  <p>
    4500
     - 
    345600000
  </p>
</div>
`;
